src/plays/self-clicking-button/SelfClickingButton.js

```diff
@@ -0,0 +1,54 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import { useState, useEffect, useRef } from 'react';
+
+// WARNING: Do not change the entry componenet name
+function SelfClickingButton(props) {
+  // Your Code Start below.
+  const buttonRef = useRef(null);
+  const [counter, setCounter] = useState(0);
+
+  function clickHandler(event) {
+    setCounter((prev) => prev + 1);
+  }
+
+  useEffect(() => {
+    buttonRef.current.addEventListener('click', clickHandler);
+
+    const interval = setInterval(() => {
+      buttonRef.current.click();
+    }, 2000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, []);
+
+  return (
+    <>
+      <div className="play-details">
```

Comment:
any reason for not using tailwind?

Reply:
This play is targeting a client video where we are not using Tailwind. Creating it as an example to show in it.